src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -270,7 +270,17 @@ public async Task InitializeAsync(CancellationToken cancellationToken = default)
 
                 if (!_environment.IsPlaceholderModeEnabled())
                 {
-                    _metricsLogger.LogEvent(string.Format(MetricEventNames.HostStartupRuntimeLanguage, _workerRuntime));
+                    string runtimeStack = _workerRuntime;
+
+                    // Appending the runtime version is currently only enabled for linux consumption. This will be eventually enabled for
+                    // Windows Consumption as well.
+                    if (_environment.IsLinuxConsumption())
```

Comment:
Can we do this more generically? So check for FunctionsWorkerRuntimeVersionSettingName and then if it's present append?

Reply:
Done